cpp/src/gandiva/precompiled/string_ops_test.cc

```diff
@@ -1088,4 +1088,53 @@ TEST(TestStringOps, TestSplitPart) {
   EXPECT_EQ(std::string(out_str, out_len), "ååçåå");
 }
 
+TEST(TestStringOps, TestToHex) {
+  gandiva::ExecutionContext ctx;
+  uint64_t ctx_ptr = reinterpret_cast<gdv_int64>(&ctx);
+  gdv_int32 out_len = 0;
+  gdv_int32 in_len = 0;
+  const char* out_str;
+
+  in_len = 10;
+  out_str = to_hex_binary(ctx_ptr, "TestString", in_len, &out_len);
```

Comment:
These tests are not clear. Why not use an explicit byte array like char array[] = { 0xB0, 0x11, 0xA2 }

Reply:
Changed the tests